src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -68,6 +77,9 @@ class SidebarScreen extends Component {
         this.setState({
             isCreateMenuActive: true,
         });
+        if (this.props.afterShowCreateMenu) {
+            this.props.afterShowCreateMenu(this);
```

Comment:
I would recommend removing `this` from the parameters. Instead pass a `ref` in the `SidebarScreen/index.js`